recipes/codan/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0" %}
+
+package:
+  name: codan
+  version: {{ version }}
+
+source:
+  url: https://github.com/pedronachtigall/CodAn/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 561d212b5acf8201dfa1af51688a6d9b0e7b8d97c1c328571987b4dda6369147
+
+requirements:
+  build:
```

Comment:
This (the build block) should be moved toplevel. There can be a build section under requirements, but this should list build requirements.

Reply:
@bernt-matthias now the lint passed, but we are having problems in the linux-test. How can I check the output of CI system without logging into CI?

Reply:
You need to login. But there should be a button "login with github" which simplifies this.